### ISSUE TITLE
Add flex Box and flex direction properties

### ISSUE THREAD

**Issue by altavir** [subroh0508/kotlin-material-ui#28]

**Comment by subroh0508** [comment]
Is the relevant API documentation here?

https://material-ui.com/system/flexbox/#flexbox

**Comment by altavir** [comment]
Yes, this one. I managed to do that via css handles, but it would be nice to be able to do it via properties.

**Comment by altavir** [comment]
There are also spacing and sizing options which would be nice to have.
